templates/include/footer.html

```diff
@@ -85,65 +85,28 @@ <h6 class="ds-h4">Additional resources</h6>
-                    <path d="M25.2951866,3.50332396 C24.4638894,2.67202675 25.2358082,0.890675584 26.4233756,0.890675584 C27.1359161,0.890675584 27.907835,1.66259442 27.907835,2.37513489 C27.907835,3.56270233 26.1264838,4.33462117 25.2951866,3.50332396 Z M25.8295919,2.31575652 C25.8295919,1.63290524 25.7405244,1.45477012 25.5030109,1.69228361 C25.3248758,1.87041873 25.2654974,2.28606733 25.3842541,2.64233756 C25.6514568,3.53301315 25.8295919,3.4142564 25.8295919,2.31575652 Z M26.9280918,2.49389163 C26.8390243,2.40482408 26.5718216,2.37513489 26.3639973,2.46420245 C26.1264838,2.55327001 26.2155513,2.64233756 26.5421324,2.64233756 C26.8687134,2.67202675 27.0468486,2.58295919 26.9280918,2.49389163 Z M27.5218755,2.01886466 C27.432808,1.78135117 27.3437404,1.87041873 27.3437404,2.19699977 C27.3140512,2.52358082 27.4031188,2.70171594 27.4921863,2.58295919 C27.5812539,2.49389163 27.6109431,2.22668896 27.5218755,2.01886466 Z M26.8687134,1.48445931 C26.7796459,1.33601338 26.5718216,1.18756744 26.3936865,1.18756744 C26.2452405,1.18756744 26.1264838,1.33601338 26.1264838,1.48445931 C26.1264838,1.66259442 26.3343081,1.78135117 26.6015108,1.78135117 C26.8390243,1.78135117 26.957781,1.66259442 26.8687134,1.48445931 Z" id="Shape"></path>
-                </g>
-            </g>
-        </g>
-    </g>
-</svg>
+<div class="sub-footer-message-container">
+  {% if user.is_authenticated %}
+  <div class="disclosure-message">
+    {{ settings.APPLICATION_TITLE }} is operated by {{settings.ORGANIZATION_TITLE }}. {{settings.DISCLOSURE_TEXT}}
+  </div>
+  {% endif %}
 
-      </a>
+  {% if user.is_superuser %}
```

Comment:
This is in no way necessary but since you're here could you just remove this? It looks like old debug code that never got removed

Reply:
Fixed in commit below!